manual.py

```diff
@@ -0,0 +1,568 @@
+        ingestion_models.EventMinutesItem(
+            minutes_item = ingestion_models.MinutesItem("RECOMMENDATION from Director Houston Public Works for approval of final contract amount of $2,949,927.38 and acceptance of work on contract with REYTEC CONSTRUCTION RESOURCES, INC for FY2018 Drainage Rehab Work Orders #4 - 1.67% under the original Contract Amount - DISTRICTS A - PECK; B - JACKSON; C - KAMIN; D - EVANS-SHABAZZ; E - MARTIN; G - HUFFMAN; H - CISNEROS; I - GALLEGOS and K - CASTEX-TATUM")
+        ),
+        ingestion_models.EventMinutesItem(
+            minutes_item = ingestion_models.MinutesItem("RECOMMENDATION from Director Houston Public Works for approval of final contract amount of $2,979,476.73 and acceptance of work on contract with GRAVA LLC for FY2019 Open Drainage System Maintenance Work Orders #1 - 0.68% under the original Contract Amount - DISTRICTS A - PECK; B - JACKSON; C - KAMIN; D - EVANS-SHABAZZ; E - MARTIN; G - HUFFMAN; H - CISNEROS; I - GALLEGOS; J - POLLARD and K - CASTEX-TATUM")
+        ),
+        ingestion_models.EventMinutesItem(
+            minutes_item = ingestion_models.MinutesItem("PROPERTY - NUMBER 3")
+        ),
+        ingestion_models.EventMinutesItem(
+            minutes_item = ingestion_models.MinutesItem("PURCHASING and TABULATION OF BIDS - NUMBERS 4 through 9")
+        ),
+        ingestion_models.EventMinutesItem(
+            minutes_item = ingestion_models.MinutesItem("ORDINANCES - NUMBERS 10 through 26"),
+            matter = #How to add more matters
+                ingestion_models.Matter(
+                "ORDINANCE approving and authorizing third amendment to contract between City of Houston and BRENTWOOD ECONOMIC COMMUNITY DEVELOPMENT CORPORATION, d/b/a BRENTWOOD COMMUNITY FOUNDATION, to extend term of contract and provide additional Housing Opportunities for Persons With AIDS Funds for the continuing administration and operation of a Community Residence and an Emergency Rental Assistance Program with Supportive Services - $892,634.00 - Grant Fund - DISTRICT K - CASTEX-TATUM", 
+                matter_type = "ORDINANCES - NUMBERS 10 through 26",
+                # title??
+                title = "ORDINANCE approving and authorizing Subrecipient Agreement between City of Houston and CATHOLIC CHARITIES OF THE ARCHDIOCESE OF GALVESTON-HOUSTON to provide Emergency Solutions Grant (ESG), and Emergency Solutions Grant Coronavirus (ESG-CV) Funds for administration and related services in connection with the City’s Emergency Rental Assistance Program, for households that have been affected by COVID-19 - $3,058,423.79 - Grant Fund"
+                ),
+            decision = "ADOPTED",
+            votes = [
+            ingestion_models.Vote(
+                person = ingestion_models.Person(
+                    "Amy Peck",
+                    is_active = "true",
+                    # only has seat/role, what to do when only has role??
```

Comment:
Seat information is good. I found their role info on a different page but we may need to manually add that in by basically having a static JSON file to store all that info and the scraper looks up that info by the persons name for example. The page I found their roles is here: http://www.houstontx.gov/council/a/index.html

Or here: http://www.houstontx.gov/council/committees/

The second one looks decently easy to scrape.

But now I am a bit confused. Do they only record full council meetings?